src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const page = document.querySelector('body');
+  const newElement = document.createElement('div');
+  const messageTitle = document.createElement('h2');
+  const messageParagraph = document.createElement('p');
+
+  newElement.className = `notification ${type}`;
+  newElement.style.position = 'absolute';
```

Comment:
notification class already has an absolute position

Reply:
Yeah,of course)) thanks